tests/auto/tst_declarativetabmodel/tst_declarativetabmodel.cpp

```diff
@@ -58,8 +58,6 @@ private slots:
     void invalidTabs_data();
```

Comment:
At least activateTabs and closeActiveTab are failing due to signal prototype change.

QSignalSpy::QSignalSpy:88 - QSignalSpy: No such signal: 'activeTabChanged(int,int)'